backend/src/src-licenses/src/main/java/com/siemens/sw360/licenses/db/LicenseDatabaseHandler.java

```diff
@@ -370,7 +372,10 @@ public RequestStatus updateLicense(License inputLicense, User user, User request
 
             License dbLicense = new License();
             if(inputLicense.isSetId()) {
-                dbLicense = licenseRepository.get(inputLicense.getId());
+                License dbLicenseTmp = licenseRepository.get(inputLicense.getId());
```

Comment:
Well, it is intended to distinguish between the case where the id is set only because it is now equal to the shortname, but the license is new and not in the database already and the case where it is. But true, you do not need the temporary variable for that, since the dbLicense does not contain information yet. I'll rewrite it.